docker/docker-compose.yml

```diff
@@ -50,12 +50,15 @@ services:
     command: npm run start:server
     ports:
       - "1337:1337"
+    working_dir: /cuttle
     volumes: *app-volume
     depends_on:
       - database
 
   client:
     container_name: client
+    # Required to be able to hit http://client:8080 from the Cypress docker container
+    hostname: client
```

Comment:
I'm not sold on this yet, it's nice to have for image<>image communication, so probably worth leaving. May also just add it to the e2e override file.